server.py

```diff
@@ -69,6 +70,7 @@ def main(services, config):
                         help='remove object_store on start')
     args = parser.parse_args()
```

Comment:
shouldn't this live in auto-installer.sh?

Reply:
I was trying to avoid setting environment variables system wide or for a specific shell type (bash, zsh, etc.). If we set them during the auto-installer.sh process (not system wide), they are dropped by the time you start CALDERA

Reply:
Alex and I have a new approach that we're going to push up in a just a minute